mariadb_kernel/client_config.py

```diff
@@ -70,12 +79,28 @@ def _config_path(self):
 
     def get_args(self):
         rv = ""
-        keys = ["user", "host", "port", "password"]
+        keys = ["user", "host", "port", "password", "socket"]
         for key in keys:
             value = self.default_config[key]
             rv += f"--{key}={value} "
         return rv
 
+    def get_server_args(self):
+        rv = []
+        rv.extend(self.default_config["extra_server_config"])
+        # get_args return a string and we need it as a list
+        # we also do not want user as sql user might differ from exec user
```

Comment:
If you refeer to the sql user part, that is partly a leftover from a previous version where the server config always matched the client config. 
However, in this version I scrapped that since the client and server config variables do not match (eg the bind-address and host) and we cannot assume the user the client connect to the sql db with (ie root) is the user the server should be executed as (jovyan in our case).

Reply:
Modifed the comment in the lasted commit, see if that is clearer